doc/source/cookbook.rst

```diff
@@ -533,10 +537,10 @@ Unlike agg, apply's callable is passed a sub-DataFrame which gives you access to
 
    S = pd.Series([i / 100.0 for i in range(1, 11)])
 
-   def CumRet(x, y):
+   def cumRet(x, y):
```

Comment:
`cum_ret` would be the Pythonic way to name the function.

Can you also replace the names in the places where the function names are used below.